mne/channels/montage.py

```diff
@@ -175,6 +145,49 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
     Montages can contain fiducial points in addition to electrode
     locations, e.g. ``biosemi-64`` contains 67 total channels.
```

Comment:
this example uses an invalid option, ``biosemi-64``. should use ``biosemi64`` instead